mobileu/settings.py

```diff
@@ -194,3 +194,12 @@
     from oneplusmvp.settings import OPEN_SCHOOL
 except ImportError as e:
     OPEN_SCHOOL = "Open School"
+
+# Junebug config
+JUNEBUG_BASE_URL = os.environ.get("JUNEBUG_BASE_URL", "https://junebug.qa.dig-it.me/jb/channels/")
+JUNEBUG_CHANNEL_ID = os.environ.get("JUNEBUG_CHANNEL_ID", "ab79e9c5-7d15-440f-8bd0-6a0e2d2445b1/")
+JUNEBUG_ACCOUNT_NUMBER = os.environ.get("JUNEBUG_ACCOUNT_NUMBER", "")
+JUNEBUG_FAKE = "*11910"
+JUNEBUG_FAKE_TO = "0660000000"
+JUNEBUG_USERNAME = "junebug"
+JUNEBUG_PASSWORD = "DbAOTBsm4zXbU10x"
```

Comment:
@FredJEhlers This is unacceptable. You can't check in passwords into a public repository. I've going to revoke these credentials. Please rather leave these blank.

Secondly, you also need to ensure that these values are read from ENV.